packages/block/src/header.ts

```diff
@@ -414,7 +414,7 @@ export class BlockHeader {
    * Checks if the block header is a genesis header.
    */
   isGenesis(): boolean {
-    return this.number.equals(toBuffer(0))
+    return this.number.equals(zeros(0))
```

Comment:
I don't understand how this has not lead to problems in the past; `toBuffer(0)` is `Buffer< 00 >` but the unpadded buffer for the genesis block is thus the empty buffer (`Buffer<>`)

Reply:
this would also be easier as a BN, `this.number.isZero()`